lms/templates/courseware/progress.html

```diff
@@ -104,6 +105,49 @@ <h2 class="title">${_("Congratulations, you qualified for a certificate!")}</h2>
         <div class="grade-detail-graph" id="grade-detail-graph" aria-hidden="true"></div>
       %endif
 
+      %if credit_course is not None:
+        <div class="credit-eligibility">
+            <div class="credit-eligibility-container">
+                <div class="eligibility-heading">
+                    <h2>${_("Eligibility Requirements for credit")}</h2>
+                </div>
+                %if credit_course['eligibility_status'] == 'not_eligible':
+                    <span class="eligibility_msg">${student.username}, ${_("You are no longer eligible for this course")}</span>
```

Comment:
you are no longer eligible for credit in this course. (note capitalization - the sentence starts with "student.username", so "you" shouldn't be capitalized)